app/platform/vfs.c

```diff
@@ -420,14 +419,14 @@ void vfs_clearerr( const char *name )
 
 #ifdef BUILD_SPIFFS
   if (fs_fns = myspiffs_realm( normname, &outname, FALSE )) {
-    fs_fns->clearerr( );
+    //fs_fns->clearerr( );
```

Comment:
This looks like it might be a functional change, in contrast to much of the rest of this PR?

Reply:
Yikes! That one snuck in by mistake! Thanks for picking it up! Can you also enlighten me why the compiler throws a
```
vfs.c: In function 'vfs_clearerr':
vfs.c:422:13: error: expected identifier before '(' token
     fs_fns->clearerr( );
             ^
make[2]: *** [../../Makefile:416: .output/eagle/debug/obj/vfs.o] Error 1
```
at me when it's not commented out?
Changing it to:
```
   void (*wtf)(void) = fs_fns->clearerr;
    wtf();
```
does not cause an error.
I'm too tired to see what's up with this...

Reply:
Ah, it's gonna be a macro thing, isn't it?

Reply:
Go to BED. I'll take a look.

Reply:
Workaround pushed.

Reply:
Alright, I'm gonna take Terry's suggestion and hit the sack now. A more elegant workaround would certainly not go astray.